src/Scanner/MethodScanner.php

```diff
@@ -538,6 +538,8 @@ protected function scan()
 
             case T_VARIABLE:
             case T_STRING:
+            case T_NAME_QUALIFIED:
```

Comment:
Where should I put the backword compatibility definitions for the new constants?

Reply:
Anywhere you want, as long as it's `@internal`, I suppose?

Reply:
Analyzing the source, `\Laminas\Code\Scanner\TokenArrayScanner::scan` seems the right place: https://github.com/laminas/laminas-code/pull/47/files#diff-c08ce1832ae9f8788d45e1cf8fa62d743ede600c44a4e622ec20b4b684383320R383
Had a run with `--process-isolation`, and everything works fine.